src/app/shared/auth-guard.service.ts

```diff
@@ -17,10 +17,7 @@ export class AuthService {
       .get('_session', { withCredentials: true })
       .pipe(switchMap((res: any) => {
         if (res.userCtx.name) {
-          // If user already matches one on the user service, do not make additional call to CouchDB
-          if (res.userCtx.name === this.userService.get().name) {
-            return of(true);
-          }
+          console.log("Res", res.userCtx.name)

```

Comment:
This code stops unnecessary GET requests to the CouchDB when we already have the user information.  Please modify this to not run setProfile if it is already set